front/poke/temporal/workflows.ts

```diff
@@ -6,10 +6,49 @@ const { scrubDataSourceActivity } = proxyActivities<typeof activities>({
   startToCloseTimeout: "60 minute",
 });
 
+const { isWorkflowDeletableActivity } = proxyActivities<typeof activities>({
```

Comment:
you can import all of these in a single `proxyActivityes` call

Reply:
Done! 👍🏻